.circleci/config.yml

```diff
@@ -156,6 +156,12 @@ run_pipe_benchmark: &run_pipe_benchmark
       command: |
         python benchmarks/pipe.py
 
+run_moee_benchmark: &run_moe_benchmark
```

Comment:
typo: extra e?

Reply:
Thx! Remove the MOE benchmark test from the circleCI.